src/sentry/workflow_engine/endpoints/validators/base/data_condition_group.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+
+from sentry.api.serializers.rest_framework import CamelSnakeSerializer
+from sentry.workflow_engine.endpoints.validators.base import BaseDataConditionValidator
+
+
+class BaseDataConditionGroupValidator(CamelSnakeSerializer):
+    logic_type = serializers.CharField(required=True)
+    organization_id = serializers.IntegerField(required=True)
```

Comment:
do we need to validate that this is a real organization id? i think project/org is usually left out of the serializer because we validate the project/org based on the url path param, and we pass it as `context` instead https://github.com/getsentry/sentry/blob/9090ac8196457a141e826563ed709a6f01690b90/src/sentry/api/endpoints/project_rules.py#L753-L755

Reply:
👍 yeah, i saw some of those too. can update to follow that pattern. (there are so many patterns for these in our codebase it's hard to tell what is "right" 😅)

Reply:
Yeah, that's a common pattern, although we could also build an `OrganizationField` or something if we need to validate in this class